docs/_pages/releases.md

```diff
@@ -11,6 +11,7 @@ sidebar:
 
 ### What's New
 * Adding `ThatAreStatic()` and `ThatAreNotStatic()` for filtering in method assertions - [#1740](https://github.com/fluentassertions/fluentassertions/pull/1740)
+* Adding new assertions for the `HttpStatusCode` of an `HttpResponseMessage` - [#1737](https://github.com/fluentassertions/fluentassertions/pull/1737)
```

Comment:
How about adding a hint for the new reference (in net47) to `System.Net.Http`?

Reply:
You mean in the release notes?

Reply:
Yes. For 6.X.Y, there will not be any migration guide, isn't it?

Reply:
No, because there's nothing to migrate.

Reply:
Then this would be the only place to inform the user.
This is not a breaking change but in an environment of restricted nuget access (like my company's package filter) it would be good to know before.

Reply:
Hmm, that's kind of edgy case. Are you saying you have to get approval to take a dependency on Microsoft packages? If so, feel free to propose a PR that captures this.